packages/lb-annotation/src/core/pointCloud/store/index.ts

```diff
@@ -719,6 +722,20 @@ class PointCloudStore {
   public setAttribute(attribute: string) {
     this.currentAttribute = attribute;
   }
+
+  public setSubAttribute(key: string, value: string) {
+    if (!this.cacheSegData) {
+      return;
+    }
+    this.cacheSegData.subAttribute = {
```

Comment:
This line needs to be compatible with the original data where `subAttribute` may not exist